### PR TITLE
User can view a list of anime 1

### PR DESCRIPTION
**Pull request by kaitlin-lovrich**
Animation

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
+body {
+  line-height: 2;
+  font-size: 0.9rem;
+  text-shadow: 0 2px 2px rgb(0, 0, 0, 0.75);
+  font-family: "Source Sans 3", sans-serif;
+  letter-spacing: 0.03rem;
+
+  /* min-height: 100vh; */
```
Comment (shawnkost) [inline css/styles.css]: Delete all old code

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  $div.appendChild($span);
+  return $li;
+}
+
+function populateRow(urlQuery, genreOrCategoryName) {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', urlQuery);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    // console.log('xhr.response:', xhr.response);
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+function getGenreId() {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://api.jikan.moe/v4/genres/anime?filter=genres');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', async function () {
+    function delay(ms) {
+      return new Promise(resolve => setTimeout(resolve, ms));
+    }
+    const createUser = id => {
```
Comment (shawnkost) [inline js/main.js]: I would rename this function. `getGenre` instead or something

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      return fetch(
+        `https://api.jikan.moe/v4/anime?order_by=popularity&genres=${id}`
+      );
+    };
+    async function runPromises() {
+      for (const genre of xhr.response.data) {
+        // console.log('genre', genre);
```
Comment (shawnkost) [inline js/main.js]: Remove all old code

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        $titles.appendChild($div);
+        $div.appendChild($i);
+
+        for (const title of genreID.data) {
+          const $titleImage = renderTitles(title);
+          $ul.appendChild($titleImage);
+        }
+        $div.appendChild($ul);
+        $div.appendChild($i2);
+        // console.log('genreID', genreID);
```
Comment (shawnkost) [inline js/main.js]: Remove all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Kaitlin!

File: js/main.js [hunk 6]
```diff
@@ -52,7 +51,7 @@ function getGenreId() {
   xhr.open('GET', 'https://api.jikan.moe/v4/genres/anime?filter=genres');
   xhr.responseType = 'json';
   xhr.addEventListener('load', async function () {
-    function delay(ms) {
+    function delayGetGenreID(ms) {
       return new Promise(resolve => setTimeout(resolve, ms));
     }
     const createUser = id => {
```
Comment (shawnkost) [inline js/main.js]: This still wasn't addressed. This should be renamed to something like `getGenre` rather than `createUser`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here Kaitlin!

File: js/main.js (hunk 6 above)
Comment (kaitlin-lovrich) [reply js/main.js]: Oh that function! Renaming the delay function didnt make alot of sense to me lol. Can do!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**